unfoldingWord/translationCore
# Refactor of Load Methods

#### This pull request addresses:
This PR completely refactors the loading process for our new data structure
Write a few sentences about what your pull request addresses, the relevant issue numbers, etc.



#### How to test this pull request:
Open a project and see if the props in the console are available from project loaded


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/956)
